views/index.html

```diff
@@ -32,6 +32,135 @@ <h1 class="name">Hector Torres</h1>
             </section>
             <img class="footer-image" src="../images/image_geometry_2.png" alt="footer image">
         </section>
+        <section class="projects">
+            <h1 class="name intro"> Projects </h1>
+            <article>
+                <img class="project-picture" src="../images/Rectangle_21.png" alt="project1">
```

Comment:
- [ ] Please ensure that the background image is added in the CSS file instead of adding them as normal `<img>` tags.